DataScience/GpxWavLabeler.py

```diff
@@ -0,0 +1,501 @@
+from datetime import datetime
+import math
+from pathlib import Path
+
+from lxml import etree
+from matplotlib import pyplot as plt
+import numpy as np
+import pandas as pd
+from pydub import AudioSegment
+from sklearn.cluster import KMeans
+
+
+# WGS84 parameters
+R_OPLUS = 6378137  # [m]
+F_INV = 298.257223563
+
+# Clip parameters
+DELTA_T_MAX = 4
+
+# Data and clip directories
+DATA_HOME = Path(
```

Comment:
Would it be possible to make these into cmd parameters? argc/argv is okay unless ArgumentParser is trivial

Reply:
Sure.